src/pages/build/deployment-form/deployment-form.jsx

```diff
@@ -9,11 +9,11 @@ import css from './deployment-form.module.scss';
 
 const cx = classNames.bind(css);
 
-const DeploymentForm = ({ handleSubmit, handleCancel }) => {
+const DeploymentForm = ({ handleSubmit, handleCancel, action, target, commit, parameters }) => {
```

Comment:
Is the `commit` prop ever used?

Reply:
ah good catch. This was copied over from the repo.jsx. I removed it from both, since it is not a used query param in either of the build/promote modals

Reply:
sounds good - thank you :)